internal/servers/controller/auth/auth.go

```diff
@@ -67,6 +69,39 @@ type RequestInfo struct {
 	DisableAuthEntirely  bool
 }
 
+// ToPb will convert a RequestInfo to a protobuf version that can be serialized
+// across the wire to the grpc-gateway server.
+func (r *RequestInfo) ToPb() *authpb.RequestInfo {
```

Comment:
Rather than do conversions, should we just remove the RequestInfo struct in this file and only use the pb message everywhere?

Reply:
done.